osf_tests/factories.py

```diff
@@ -1115,3 +1115,54 @@ class Meta:
 
     primary_color = factory.Faker('hex_color')
     secondary_color = factory.Faker('hex_color')
+
+
+class SchemaResponsesFactory(DjangoModelFactory):
+    class Meta:
+        model = models.SchemaResponses
+
+
+class RegistrationSchemaFactory(DjangoModelFactory):
+    schema_version = factory.Sequence(lambda n: n)
+    schema = {
+        'title': 'Test Data Schema',
+        'version': 1,
+        'config': {
+            'hasFiles': True
+        },
+        'description': 'Test Schema for internal use only',
+        'pages': [{
+            'id': 'page1',
+            'title': 'Study Information',
+            'questions': [{
+                'qid': 'q1',
+                'nav': 'Nav, what does this do?',
+                'type': 'string',
+                'format': 'textarea',
+                'title': 'Maybe a section heading',
+                'description': 'Maybe a question label.',
+                'help': 'example text for question-label',
+                'required': True
+            }, {
+                'qid': 'q2',
+                'nav': 'Hypotheses',
+                'type': 'string',
+                'format': 'textarea',
+                'title': 'Maybe a section heading',
+                'description': 'Maybe a question label.',
+                'help': 'help text',
+                'required': True
+            }]
+        }]
+    }
+
+    @classmethod
+    def _create(cls, *args, **kwargs):
```

Comment:
add some logic here to make this a singleton, since it's a canned default?

Reply:
I don't really understand what you're asking.

Reply:
This is a canned schema. Every call to this factory will produce the same thing, minus the ID. You can safely check to see if a schema already exists with `schema['title']` and just return that schema if it exists instead of creating duplicates.

In general, this feels more like a `DefaultTestSchema` that maybe belongs in.a shared fixture than a `SchemaFactory`